lib/doorkeeper/engine.rb

```diff
@@ -5,7 +5,9 @@ class Engine < Rails::Engine
     end
 
     initializer "doorkeeper.locales" do |app|
-      app.config.i18n.fallbacks = [:en]
+      if app.config.i18n.fallbacks.blank?
+        app.config.i18n.fallbacks = [:en]
+      end
     end
```

Comment:
I couldn't find Rails documentation on the `fallbacks` option. When not set, in a Rails application I see it as an empty hash (it's in fact an `ActiveSupport::OrderedOptions`, a small subclass of `Hash`), which is close to what the i18n rubygem defines (https://github.com/svenfuchs/i18n/blob/cb679b8cdbab675703a3f88de4d48a48f7b50e06/lib/i18n/locale/fallbacks.rb). A code search in GitHub shows many projects using it as a boolean?

What's the correct way of adding a general default to English, when none is set? Even if it works with current implementation, I'd like to make sure we are using and setting the expected APIs (either by Rails or the i18n project).